lib/src/core/provider/user_provider.dart

```diff
@@ -28,7 +29,9 @@ class UserProvider with ChangeNotifier {
   }
 
   Future<bool> checkIsAppleSignInAvailable() async {
-    isIphone = await SignInWithApple.isAvailable();
+    if (!kIsWeb) {
+      isIphone = await SignInWithApple.isAvailable();
+    }
```

Comment:
i think it would be better to do like this `if(kIsWeb) return false;` and leave previous check as it is